### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by UebelAndre**
This is work pulled out of #276 

Changes:
1. renamed `settings::testing` to `settings::tests` to be consistent with other modules
2. Ran `cargo fmt`

### CODE REVIEW

**Review by GregBowyer — APPROVED** [review]

File: impl/src/bin/cargo-raze.rs [hunk 1]
```diff
@@ -121,10 +121,13 @@ fn main() -> Result<()> {
     GenMode::Remote => {
       if !dry_run {
         let remote_dir = render_details.path_prefix.as_path().join("remote");
-        
+
         // Clean out the "remote" directory and guarantee that it exists
         if remote_dir.exists() {
-          for entry in glob::glob(&format!("{}/BUILD*.bazel", &remote_dir.display().to_string()))? {
+          for entry in glob::glob(&format!(
```
[the remark this answers is not in the document]
Comment (UebelAndre) [reply impl/src/bin/cargo-raze.rs]: Updated. I might have misread what you wrote, I stuck with `std::path::Display` but are you saying `to_string_lossy` is better?

**Comment by UebelAndre** [comment]
@GregBowyer this is ready for another review

File: impl/src/bin/cargo-raze.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (GregBowyer) [reply impl/src/bin/cargo-raze.rs]: No I was more suggesting options

**Comment by GregBowyer** [comment]
:+1: LGTM

**Comment by UebelAndre** [comment]
@acmcarther this PR is ready for review

**Review by dfreese — APPROVED** [review]
Looks fine to me, but can you please update the PR to be more specific about both what is happening, and why?

**Comment by UebelAndre** [comment]
@dfreese done

**Pull request merged**